src/utils/geoJsonUtils.js

```diff
@@ -77,3 +81,13 @@ export function transformIntoTurfEquivalent(geoJsonData, fromProjection = WGS84)
     log.error('Unknown geometry type', geometryWGS84.type)
     return null
 }
+
+/**
+ * @param {Object[]} geometries An array of all geometries in features.
+ * @returns An extent which covers all geometries given as parameter
```

Comment:
you could specify that the returned type is `[[Number,Number], [Number,Number]]`

sometimes extents are described inline (`[a,b,c,d]`) but here we intentionally normalized it to `[[a,b], [c,d]]`